calender/calendar_tui.py

```
@@ -1254,6 +1258,7 @@ def draw_attendee_details(self):
                 no_response.append(attendee_info)
 
         total = len(event.attendees)
+        content_width = modal_width - 4  # usable width inside borders + padding
```

Ruff (0.15.1)

[error] Local variable `content_width` is assigned to but never used

Remove assignment to unused variable `content_width`

(F841)